src/fonts/museo.scss

```diff
@@ -6,60 +6,60 @@
 
 @font-face {
   font-family: 'MuseoSansRounded100Regular';
-  src: url('../fonts/museo/MuseoSansRounded-100-webfont.eot');
-  src: url('../fonts/museo/MuseoSansRounded-100-webfont.woff') format('woff'),
-    url('../fonts/museo/MuseoSansRounded-100-webfont.ttf') format('truetype'),
-    url('../fonts/museo/MuseoSansRounded-100-webfont.svg') format('svg');
+  src: url('/assets/fonts/museo/MuseoSansRounded-100-webfont.eot');
+  src: url('/assets/fonts/museo/MuseoSansRounded-100-webfont.woff') format('woff'),
```

Comment:
I still don't know what the frack I'm doing with fonts. I think these need something other than a relative path to work in production, locally, and referenced from different apps, so we either can use the assets from Classic or reference what will be published to CDN from here. I'm deferring this question for now and figure we can experiment once these are up on CDN.